assembl/lib/antivirus.py

```diff
@@ -0,0 +1,69 @@
+from tempfile import mkdtemp
+from subprocess import call
+from os import unlink, rmdir
+from os.path import join
+from random import random
+
+from pyramid.path import DottedNameResolver
+
+from . import config
+
+
+resolver = DottedNameResolver(__package__)
+
+
+class AntiVirus(object):
+    def check(self, data, suffix=None):
+        return NotImplementedError()
+
+
+class MockPositiveAntiVirus(AntiVirus):
```

Comment:
TODO: @mohamedzayed92 Should add these to the list of entities to be tested as there are no unit tests in this PR.

Reply:
no problem @ayyazdaniaryan ... As soon as it gets merged, i will create a fixture for MockPositiveAntivirus and unit tests